### PR TITLE
Removed StatusBullet, TimerPulser, DatePickerRange, DatePickerInputRange

### PR DESCRIPTION
**Pull request by qubis741**
https://teamleader.atlassian.net/browse/ATL-722

### Description

Removed StatusBullet, TimerPulser, DatePickerRange, DatePickerInputRange

### Breaking changes

-  removed StatusBullet, TimerPulser, DatePickerRange, DatePickerInputRange

### CODE REVIEW

**Review by joaoinacio — APPROVED** [review]

**Review by ArnaudWeyts — APPROVED** [review]
Removing the components could have been separate commits, other than that, looks good 👌🏻

File: src/components/datepicker/DatePickerRange.js [hunk 1]
```diff
@@ -1,130 +0,0 @@
...
-    const { selectedStartDate, mouseEnteredEndDate } = this.state;
-    const modifiers = { from: selectedStartDate, to: mouseEnteredEndDate };
-    const selectedDays = [selectedStartDate, { from: selectedStartDate, to: mouseEnteredEndDate }];
-
-    const boxProps = pickBoxProps(others);
-    const restProps = omitBoxProps(others);
-    const classNames = cx(
-      uiUtilities['reset-font-smoothing'],
-      theme['date-picker'],
-      theme['has-range'],
```
Comment (ArnaudWeyts) [inline src/components/datepicker/DatePickerRange.js:84]: I think this css class can also be removed

**Review by ArnaudWeyts — CHANGES_REQUESTED** [review]
Actually, I think we can also remove the `DatePickerInputRange`, while we are at it, wasn't listed in the ticket but it's related (don't forget to also double check the class usages in there)

**Review by ArnaudWeyts — APPROVED** [review]

**Comment by ArnaudWeyts** [comment]
🔴 The stories for the regular date input don't seem to load anymore

**Review by ArnaudWeyts — APPROVED** [review]

**Pull request merged**